sdk/maps/azure-maps-elevation/src/main/java/com/azure/maps/elevation/implementation/helpers/Utility.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.maps.elevation.implementation.helpers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.azure.core.models.GeoBoundingBox;
+import com.azure.core.models.GeoPosition;
+import com.azure.maps.elevation.implementation.models.LatLongPairAbbreviated;
+
+public class Utility {
+    
+    public static List<LatLongPairAbbreviated> toLatLongPairAbbreviated(List<GeoPosition> points) {
+        List<LatLongPairAbbreviated> latLongPairList = new ArrayList<>();
```

Comment:
nit: create the `ArrayList` with a size equal to the number of points

`new ArrayList<>(points.size())`